fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastore.java

```diff
@@ -118,6 +119,17 @@ public Table getTable(String dbName, String tableName) {
             if (table.getTableType().equalsIgnoreCase("VIRTUAL_VIEW")) {
                 return HiveMetastoreApiConverter.toHiveView(table, catalogName);
             } else {
+                if (isHiveTableWithAvroSchemas(table)) {
```

Comment:
`getTable` is the most frequently called interface. Is there any other way to check if it's an avro table? Or sort out this part of the code. there are too many branches in this part, and it's not very clear.